dbt/include/trino/macros/materializations/incremental/merge.sql

```diff
@@ -0,0 +1,29 @@
+{% macro trino__get_delete_insert_merge_sql(target, source, unique_key, dest_columns) -%}
+    {%- set dest_cols_csv = get_quoted_csv(dest_columns | map(attribute="name")) -%}
+
+    {% if unique_key %}
+        {% if unique_key is sequence and unique_key is not string %}
+            delete from {{ target }}
+            where
+                {% for key in unique_key %}
+                    {{ target }}.{{ key }} IN (SELECT {{ key }} FROM {{ source }})
+                    {{ "and " if not loop.last }}
+                {% endfor %}
+            ;
+        {% else %}
+            delete from {{ target }}
+            where (
+                {{ unique_key }}) in (
+                select {{ unique_key }}
+                from {{ source }}
+            );
+
+        {% endif %}
+    {% endif %}
+
+    insert into {{ target }} ({{ dest_cols_csv }})
+    (
+        select {{ dest_cols_csv }}
+        from {{ source }}
```

Comment:
i'm wondering whether you missed 

```
.include(database=true, schema=true)
```

like it is done for `append`

https://github.com/starburstdata/dbt-trino/pull/62/files#diff-a5aa2b035a51111d752170921f513337ce6af7d4231b586252df7b0a7dbf71b6R30

Reply:
This is not required, it is actually already generated using database and schema. In the line after this one we can see:

`drop table if exists {{ tmp_relation }};`

If it would be necessary we should include it there too, otherwise the table would not be found.

Note that snowflake also doesn't do that.